regression/cases/splunk_hec_route_s3/vector/vector.toml

```diff
@@ -36,6 +36,7 @@ address = "0.0.0.0:9090"
 type = "aws_s3"
 inputs = ["container_type.sidecar"]
 
+region = "us-west-2"
```

Comment:
Did you find this to be needed too? Just trying to avoid the test cases falling out of sync. I think ideally we'd just have one set rather than this set in addition to `soaks/tests`. Maybe we could just symlink the test case directory for now?

Reply:
Yes, we run in a more restricted environment in the new setup. With regard to things falling out of sync, my hope is that the soak tests won't last much longer, TBH. Also if we did want to symlink we could do that but lading would need to be updated on the soak side.